client/tests/integration_tests/connected_peers.rs

```diff
@@ -0,0 +1,62 @@
+#![allow(clippy::pedantic, clippy::restriction)]
+
+use std::thread;
+
+use iroha_client::client::Client;
+use iroha_core::config::Configuration;
+use iroha_data_model::{peer::Peer as DataModelPeer, prelude::*};
+use test_network::*;
+
+#[test]
+fn connected_peers_with_f_2_1_2() {
+    connected_peers_with_f(2)
+}
+
+// TODO This case does not have to be supported, but at least have to be error-handled
+// #[test]
+// fn connected_peers_with_f_1_0_1() {
+//     connected_peers_with_f(1)
+// }
```

Comment:
you forgot about this or will it stay for future reference?

Reply:
Yes, for future reference #1716